drivers/CoreIMU/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Leka - LekaOS
```

Comment:
I suggest to give some credit to Cedric and Camelia. Please add them as co-authors.

Reply:
to add them as co-authors you need 

```
git rebase -i {sha of last merge commit} -x "GIT_EDITOR=\"echo 'Co-authored-by: NAME <NAME@EXAMPLE.COM>\nCo-authored-by: AUTHOR-NAME <ANOTHER-NAME@EXAMPLE.COM>' >>\" git commit --amend"
```

you'll also need the emails used by @CameliaB2 and @CedricDia on Github